src/types/mod.rs

```diff
@@ -4,11 +4,13 @@
 #![allow(missing_docs)] // TODO Remove this once everything has settled.
 
 /// Module containing the ledger data models.
+#[cfg(feature = "stardust")]
```

Comment:
Maybe we should do some folder reorg instead of these weird gates.

Reply:
I like that.

Reply:
Let's do this in another PR. I'll open an issue for it. Fine with that?

Reply:
Yes, fine for me.

Reply:
Created #437